package.json

```diff
@@ -49,47 +50,68 @@
     "prettier"
   ],
   "dependencies": {
+    "@typegoose/typegoose": "^9.7.1",
     "bcryptjs": "^2.4.3",
     "compression": "^1.7.4",
     "cors": "^2.8.5",
-    "cross-env": "^7.0.0",
-    "dotenv": "^10.0.0",
-    "express": "^4.17.1",
-    "express-mongo-sanitize": "^2.0.0",
-    "express-rate-limit": "^5.0.0",
-    "helmet": "^4.1.0",
-    "http-status": "^1.4.0",
-    "joi": "^17.3.0",
+    "cross-env": "^7.0.3",
+    "dotenv": "^16.0.0",
+    "express": "^4.17.3",
+    "express-mongo-sanitize": "^2.2.0",
+    "express-rate-limit": "^6.3.0",
+    "helmet": "^5.0.2",
+    "http-status": "^1.5.0",
+    "joi": "^17.6.0",
     "jsonwebtoken": "^8.5.1",
-    "moment": "^2.24.0",
-    "mongoose": "^5.7.7",
-    "morgan": "^1.9.1",
-    "nodemailer": "^6.3.1",
-    "passport": "^0.4.0",
+    "moment": "^2.29.2",
+    "mongoose": "^6.2.10",
+    "morgan": "^1.10.0",
+    "nodemailer": "^6.7.3",
+    "passport": "^0.5.2",
     "passport-jwt": "^4.0.0",
-    "pm2": "^5.1.0",
-    "swagger-jsdoc": "^6.0.8",
-    "swagger-ui-express": "^4.1.6",
-    "validator": "^13.0.0",
-    "winston": "^3.2.1",
-    "xss-clean": "^0.1.1"
+    "pm2": "^5.2.0",
+    "swagger-jsdoc": "^6.2.0",
+    "swagger-ui-express": "^4.3.0",
+    "validator": "^13.7.0",
+    "winston": "^3.7.2",
+    "xss-clean": "^0.1.1",
+    "zod": "^3.14.4"
   },
   "devDependencies": {
-    "coveralls": "^3.0.7",
-    "eslint": "^7.0.0",
-    "eslint-config-airbnb-base": "^14.0.0",
-    "eslint-config-prettier": "^8.1.0",
-    "eslint-plugin-import": "^2.18.2",
-    "eslint-plugin-jest": "^24.0.1",
-    "eslint-plugin-prettier": "^3.1.1",
+    "@faker-js/faker": "^6.1.2",
+    "@jest/globals": "^27.5.1",
+    "@types/bcryptjs": "^2.4.2",
+    "@types/compression": "^1.7.2",
+    "@types/cors": "^2.8.12",
+    "@types/express": "^4.17.13",
+    "@types/express-rate-limit": "^6.0.0",
+    "@types/jest": "^27.4.1",
+    "@types/jsonwebtoken": "^8.5.8",
+    "@types/mongoose": "^5.11.97",
```

Comment:
I think you can also delete @types/mongoose package. Mongosse publishes its own, so no need external types